CONTRIBUTING.md

```diff
@@ -299,7 +299,7 @@ maintainers. When opening a pull request, explain what the patch is doing
 and if it makes sense, why the proposed implementation was chosen.
 
 Try to use well-defined commits (one feature per commit) so that reading
-them and testing them is easier for reviewers and while bissecting the code
+them and testing them is easier for reviewers and while dissecting the code
```

Comment:
This is using git [bisect](http://www.dictionary.com/browse/bisect). The word should be corrected to 'bisecting' though, dropping the double -s